lib/srv/discovery/access_graph.go

```diff
@@ -241,7 +243,8 @@ func (s *Server) initializeAndWatchAccessGraph(ctx context.Context, reloadCh <-c
 	)
 
 	clusterFeatures := s.Config.ClusterFeatures()
-	if !clusterFeatures.AccessGraph && (clusterFeatures.Policy == nil || !clusterFeatures.Policy.Enabled) {
+	policy := modules.GetProtoEntitlement(&clusterFeatures, entitlements.Policy)
+	if !clusterFeatures.AccessGraph && !policy.Enabled {
```

Comment:
I left it mostly because its already marked as todo to remove. Seems like there will be a short term alternative.

```
	// TODO(justinas): remove this field once "TAG enabled" status is moved to a resource in the backend.
```